### PR TITLE
Runner as a service on Windows Environments

### PR DESCRIPTION
**Pull request by MegaDrive68k**
Here is the "Runner as a Service on Windows"  how to.

### CODE REVIEW

File: docs/learning/howto/runner-service-windows.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+
+Now let's see two ways of installing Automation Runner as a service.
+
+## Installing the Automation Runner service through Apache Commons Daemon
+
+Apache Commons Daemon, often referred to simply as Commons Daemon, is a set of Java libraries and utilities provided by the Apache Software Foundation. Its primary purpose is to facilitate the development of cross-platform Java applications that can be run as background services or daemons on various operating systems, including Windows.
+
+To register the Automation Runner as a service, check the following steps:
+
+1. Download Apache Commons Daemons binaries from this location.
```
Comment (fdevans) [inline docs/learning/howto/runner-service-windows.md]: There is no link here.

File: docs/learning/howto/runner-service-windows.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+
+2. Stop the service with the following command: `nssm stop "Runner"`.
+
+3. Then, remove the service with the `nssm remove "Runner" confirm` command.
+
+The service is down and unregistered from Windows Services.
+
+## Resources
+
+[Automation Runner](https://docs.rundeck.com/docs/administration/runner/pre-4-11-runners.html#runner)
```
Comment (fdevans) [inline docs/learning/howto/runner-service-windows.md]: Local links should go to the `.md` file using the path after `/docs/`. in this case:

`/administration/runner/pre-4-11-runners.md#runner`

Using a direct link doesn't allow it to work within the staged or historical version sites properly.

**Review by fdevans — CHANGES_REQUESTED** [review]

File: docs/learning/howto/runner-service-windows.md (hunk 2 above)
Comment (MegaDrive68k) [reply docs/learning/howto/runner-service-windows.md]: Changing!

**Review by jsboak — APPROVED** [review]
made changes via commits. time to ship! 🚀

**Review by fdevans — APPROVED** [review]

**Pull request merged**